project/plugins.sbt

```diff
@@ -19,7 +19,7 @@ addSbtPlugin("software.purpledragon" % "sbt-checkstyle-plugin" % "4.0.1")
 
 // If you are changing the dependency setting for checkstyle plugin,
 // please check pom.xml in the root of the source tree too.
-libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "10.17.0"
+libraryDependencies += "com.puppycrawl.tools" % "checkstyle" % "10.20.0"
 
 // checkstyle uses guava 33.1.0-jre.
 libraryDependencies += "com.google.guava" % "guava" % "33.1.0-jre"
```

Comment:
@HyukjinKwon Would like to confirm if `"com.puppycrawl.tools" % "checkstyle" % "10.20.0"` still depends on guava 33.1.0-jre?